.gitignore

```diff
@@ -1 +1,2 @@
 pre-commit-venv
+.idea/
```

Comment:
this is likely a folder that was not generated by this project but rather native to your work environment right? in that case it shouldn't be here. You are better off having this added to your personal `.git/info/exclude` (the syntax is similar to .gitignore). That way this is only found in your local env

Reply:
Thanks for this